core/shared/src/test/scala/cats/parse/ParserTest.scala

```diff
@@ -1588,6 +1588,22 @@ class ParserTest extends munit.ScalaCheckSuite {
     }
   }
 
+  property("select(pa.map(Left(_)))(pf) == pf.ap(pa)") {
+    forAll(ParserGen.gen, ParserGen.gen) { (genP, genRes) =>
+      val pa = genP.fa
+      val pf = null: Parser[genP.A => genRes.A]
```

Comment:
I'm struggling to produce this parser.  The trick used in the generator didn't quite translate.